src/views/menus/useGlobalCommands.tsx

```diff
@@ -42,83 +46,66 @@ export const useGlobalCommands = (): MenuConfig<string, string> => {
 
   return [
     {
-      group: 'themes',
-      groupLabel: 'Themes',
+      group: 'view',
+      groupLabel: stringGetter({ key: STRING_KEYS.VIEW }),
       items: [
         {
-          value: AppTheme.Classic,
-          label: 'Set Classic Theme',
-          onSelect: () => {
-            dispatch(setAppThemeSetting(AppTheme.Classic));
-          },
+          value: 'trade',
+          slotBefore: <Icon iconName={IconName.Trade} />,
+          label: stringGetter({ key: STRING_KEYS.TRADE }),
+          onSelect: () => navigate(AppRoute.Trade),
         },
         {
-          value: AppThemeSystemSetting.System,
-          label: 'Set System Theme',
-          onSelect: () => {
-            dispatch(setAppThemeSetting(AppThemeSystemSetting.System));
-          },
+          value: 'portfolio',
+          slotBefore: <Icon iconName={IconName.PriceChart} />,
+          label: stringGetter({ key: STRING_KEYS.PORTFOLIO }),
+          onSelect: () => navigate(AppRoute.Portfolio),
         },
         {
-          value: AppTheme.Light,
-          label: 'Set Light Theme',
-          onSelect: () => {
-            dispatch(setAppThemeSetting(AppTheme.Light));
-          },
+          value: 'markets',
+          slotBefore: <Icon iconName={IconName.Markets} />,
+          label: stringGetter({ key: STRING_KEYS.MARKETS }),
+          onSelect: () => navigate(AppRoute.Markets),
         },
         {
-          value: AppTheme.Dark,
-          label: 'Set Dark Theme',
-          onSelect: () => {
-            dispatch(setAppThemeSetting(AppTheme.Dark));
-          },
+          value: 'token',
+          slotBefore: <Icon iconName={IconName.Coins} />,
+          label: chainTokenLabel,
+          onSelect: () => navigate(`/${chainTokenLabel}`),
         },
+        ...(showVaults
+          ? [
+              {
+                value: 'vaults',
+                slotBefore: <Icon iconName={IconName.Governance} />,
+                label: stringGetter({ key: STRING_KEYS.VAULTS }),
+                onSelect: () => navigate(AppRoute.Vaults),
+              },
+            ]
+          : []),
```

Comment:
I saw this being done is a different way elsewhere (search for `showVaults`) but I couldn't get the syntax to work for some reason

Reply:
ah if you do `showVaults &&` you need to put a `.filter(isPresent)` at the end of the array declaration.